mixer/pkg/runtime/dispatcher/tracing.go

```diff
@@ -18,6 +18,7 @@ import (
 	rpc "github.com/gogo/googleapis/google/rpc"
 	opentracing "github.com/opentracing/opentracing-go"
 	tracelog "github.com/opentracing/opentracing-go/log"
+	"istio.io/istio/mixer/pkg/adapter"
```

Comment:
Please move this import into the group below.

Reply:
Thanks for the pointer.
I updated the PR.